src/main/java/UseCaseClasses/FriendAdder.java

```diff
@@ -12,7 +12,7 @@
      *
      */
 
-    public void sendFriendRequest(Person user1, Person user2){
+    public static void sendFriendRequest(Person user1, Person user2){
```

Comment:
this almost certainly should not be static -- did intellij suggest this?

Reply:
honestly I don't even remember adding that - so it's not necessary